src/rubric_eval/main.py

```diff
@@ -0,0 +1,281 @@
+"""
+How to run:
+
+1. git clone https://github.com/stella-z3g/RubricEval -b public_release
+2. cd RubricEval/ && pip install -e .
+3. python scripts/convert_dataset_to_rubriceval_input_format.py  # This will generate an example instructions.json
+4. rubric_eval get_rubrics --input_path=instructions.json
+5. rubric_eval get_completions --model_config=gpt-4o-2024-05-13 --input_path=instructions_with_rubrics.json
+6. rubric_eval --model_config=gpt-4o-2024-05-13 --input_path=completions.json
+"""
```

Comment:
Note to self:
- Add doctest
- Add unit tests [example](https://github.com/tatsu-lab/alpaca_eval/tree/main/tests) with mocking that are run in the CI [example](https://github.com/tatsu-lab/alpaca_eval/tree/main/.github/workflows)
- A few integration tests [example](https://github.com/tatsu-lab/alpaca_eval/tree/main/tests/integration_tests) for the main example to make sure that all the API stuff works. Run integration tests manually now, (worth running e.g. when updating pypy package)